src/scribe_data/wikidata/update_data.py

```diff
@@ -118,6 +116,38 @@ def update_data(languages=None, word_types=None):
         query_name = f"query_{target_type}.sparql"
         query_path = f"{q}/{query_name}"
 
+        # After formatting and before saving the new data
+        timestamp = datetime.now().strftime("%Y_%m_%d_%H_%M_%S")
+        export_dir = Path(f"scribe_data_json_export/{lang.capitalize()}")
+        export_dir.mkdir(parents=True, exist_ok=True)
+
+        new_file_name = f"{target_type}_{timestamp}.json"
+        new_file_path = export_dir / new_file_name
+
+        # Check for existing files with timestamps
+        existing_files = list(export_dir.glob(f"{target_type}_*.json"))
+        if existing_files:
+            print(f"Existing file(s) found for {lang} {target_type}:")
+            for i, file in enumerate(existing_files, 1):
+                print(f"{i}. {file.name}")
+
+            choice = input("Choose an option:\n1. Keep existing (skip update)\n2. Overwrite existing\n3. Keep both\n4. Cancel\nEnter your choice (1-4): ")
+
+            if choice == '1':
+                print(f"Skipping update for {lang} {target_type}")
+                continue
+            elif choice == '2':
+                for file in existing_files:
+                    file.unlink()
```

Comment:
I'm putting this down in the next notes doc :) This would be only files like this in the output directory, but I agree that this is a bit dangerous. Let's figure this out!